lib/OpenQA/Task/Needle/Delete.pm

```diff
@@ -18,7 +18,7 @@ sub _delete_needles ($app, $minion_job, $args) {
     # SignalGuard will prevent the delete task to interrupt with no recovery,
     # instead will retry once the gru server returned up and running. The popup
     # on the frontend will wait until the retried job finished.
-    my $signal_guard = OpenQA::Task::SignalGuard->new($minion_job);
+    my $signal_guard = OpenQA::Task::SignalGuard->new($minion_job, abort => 1);
```

Comment:
I don't know. I said we should set `retry` to 0 below and leave `abort` alone. In my opinion the value of `abort` is fine.
`retry` is 1 as a default, and `abort` is 0.
So now we would set `abort` to 0 below, but `retry` is still set to 1, and retry is also aborting the job, only that it retries it, so it would still have the same problem.

But @Martchus seems to have a different opinion.

Reply:
@perlpunk Sounds like you looked up how these attributes interact in detail. So I guess I trust your suggestion then :-)

Reply:
@Wabri Yes, that's what I would suggest